steps/dotnetcore/1.0.0/step.yml

```diff
@@ -0,0 +1,37 @@
+title: .NET Core
```

Comment:
Will change soon ;) - did you try the new workflow editor? https://discuss.bitrise.io/t/offline-workflow-editor-workflow-editor-v2/39